text/0006-architecture-review-process.md

```diff
@@ -114,7 +112,7 @@ In order to agree on the proposal, the reviewer and the team/technical lead or p
 
 * Establish vocabulary
 * Help the team “argue the right level” for the review
-* Identify stakeholders and domain experts — those Consulted, per RACI — in the organization
+* Identify stakeholders and domain experts — those Consulted
```

Comment:
Nit: "those Consulted" here refers specifically to the "consulted" role in the RACI model, but you've otherwise removed all references to that model from this document, so I would remove this reference as well, either by striking the expression or by incorporating the word "consulted" without the formalism of capitalization, i.e.:

* Identify the stakeholders and domain experts who will be consulted

Reply:
Fixed in a13378a.